### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by MohdShabaz** [unoplatform/uno#9330] [General]
Hi team,

We want to develop a cross-platform application. We are considering UNO to develop the application and use it for production. This application will be distributed to our customers. Do we need to get a license for it? 

Thank you,
Regards,
Shabaz

**Comment by francoistanguay** [comment]
No license required to you Uno. It is provided under the Apache 2 license as explained here: https://github.com/unoplatform/uno/blob/master/License.md

It's Free as in Free and Open-Source!

**Reply by MohdShabaz** [reply]
Thank you so much.
